src/client/js/api.js

```diff
@@ -0,0 +1,13 @@
+const BASE_URL = "";
+
+export const fetchCategory = () => fetch(`${BASE_URL}`)
+.then(res => {
+  if(res.status === 200) { // GET success status expect 200
```

Comment:
PR올리기전에 불 필요한 코드는 삭제해서 올려주시면 좋을 것 같아요( 주석이나 개발을 위해 활용했던 것들.)
```suggestion
  if (res.status === 200) { 
```
일반적으로 상태코드 보단 상태메세지 "ok" 를 활용하기도 합니다. 200, ok로 많이 상태 코드와 응답 메세지를 보내주기 때문에요